### PR TITLE
Enable to get a bound statement in ClickHousePreparedStatement

### PR DESCRIPTION
**Pull request by h6ak**
I have implemented `ClickHousePreparedStatement.toString()`.
This method returns bound statement when `buildSql()` succeeds, otherwise unbound statement.

### CODE REVIEW

File: src/main/java/ru/yandex/clickhouse/ClickHousePreparedStatement.java [hunk 1]
```diff
@@ -25,4 +25,6 @@
     ResultSet executeQuery(Map<ClickHouseQueryParam, String> additionalDBParams, List<ClickHouseExternalData> externalData) throws SQLException;
 
     int[] executeBatch(Map<ClickHouseQueryParam, String> additionalDBParams) throws SQLException;
+
+    String toString();
```
Comment (enqueue) [inline src/main/java/ru/yandex/clickhouse/ClickHousePreparedStatement.java]: This is not required in Java; the class can simply override Object#toString().

**Comment by enqueue** [comment]
@h6ak  what do you want t to achieve? Is this just for debugging purposes? `toString` is a very special method in Java, as you perhaps know.

**Comment by h6ak** [comment]
@enqueue 
Thank you for your comment.

> what do you want t to achieve? Is this just for debugging purposes?

Yes, the goal is to enable us to check bound statements in our program for debugging.

Should I rename the implemented method ?

**Comment by enqueue** [comment]
@h6ak I think this would be useful, but `toString` is called in places you do not expect / cannot control, and it might be expensive when you have statements with thousands of parameter values. I would suggest to use a different method name. If you still want to use `toString`, you can remove the method declaration from the interface `ClickHousePreparedStatement` because you will just override `Object#toString`.

**Comment by h6ak** [comment]
@enqueue I understand what you say. I've renamed the implemented method to `asSql`.

**Pull request merged**